yarn.lock

```diff
@@ -5798,7 +5798,50 @@ __metadata:
   languageName: unknown
   linkType: soft
 
-"@storybook/addon-essentials@6.3.0-beta.5, @storybook/addon-essentials@workspace:*, @storybook/addon-essentials@workspace:addons/essentials":
+"@storybook/addon-essentials@portal:/Users/varunvachhar/Documents/chromatic/storybook/addons/essentials::locator=%40storybook%2Froot%40workspace%3A.":
```

Comment:
Did you link something local to the monorepo while testing?

Reply:
I did, and I thought I removed it. Good catch.